framework/console/widgets/Table.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace yii\console\widgets;
+
+use Yii;
+use yii\base\Object;
+use yii\helpers\ArrayHelper;
+use yii\helpers\Console;
+
+/**
+ * Class Table
+ *
+ * @author Daniel Gomez Pan <pana_1990@hotmail.com>
+ * @since 2.0.12
+ */
+class Table extends Object
+{
+    private $_headers = [];
+    private $_rows = [];
+    private $_chars = [
```

Comment:
phpdoc missing. Also it's better to be formatted so each pair takes its own line.